lib/Doctrine/DBAL/Types/BlobType.php

```diff
@@ -46,7 +46,7 @@ public function convertToPHPValue($value, AbstractPlatform $platform)
         }
 
         if (is_string($value)) {
-            $fp = fopen('php://temp', 'rb+');
+            $fp = tmpfile();
```

Comment:
Hm this creates a real file, php://temp only does when over 2MB. Not sure about this.